kubernetes/gcp-gke/blueprint.yaml

```diff
@@ -0,0 +1,95 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  This blueprint create a managed cluster.
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.4/types.yaml
+  - plugin:cloudify-gcp-plugin?version= >=1.6.0
+  - plugin:cloudify-kubernetes-plugin?version= >=2.6.3
```

Comment:
Let's update this to kubernetes plugin `2.6.5` since this has the fix for the GCP authentication.